Common.Packages.props

```diff
@@ -0,0 +1,8 @@
+<Project>
```

Comment:
Why is the `Common.Packages.props` needed? I mean couldn't we have only the `Directory.Packages.props`?

Reply:
Hum I guess it is just to align with the other `Common.*` files ...

Reply:
Removed.